speed-dreams/source-2.2.3/data/tracks/simple-highway/simple-highway-4-80/simple-highway-4-80.xml

```diff
@@ -0,0 +1,1844 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+file                : Highway_with_Curves.xml
+auto generated      : by Track Editor
+version             : v 0.6.0 24/4/2005
+copyright           : (C) 2005 by Charalampos Alexopoulos
+email               : 
+-->
+<!--
+This program is free software; you can redistribute it and/or modify it
+under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+-->
+<!DOCTYPE params SYSTEM "../../../src/libs/tgf/params.dtd" [
+<!--  general definitions for tracks  -->
+<!ENTITY default-surfaces SYSTEM "../../../data/tracks/surfaces.xml">
+]>
+
+<params name="test" type="param" mode="mw">
+  <section name="Surfaces">&default-surfaces;</section>
+  <section name="Header">
+    <attstr name="name" val="4 lanes 80" />
```

Comment:
This name is actually shown in-game, so should be more clear like:
```suggestion
    <attstr name="name" val="Highway 4 Lanes [80 km/h]" />
```
Or something similar.